main.css

```diff
@@ -1,3 +1,83 @@
-body {
-  background-color: aliceblue;
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+
+nav > p {
+  left: 12px;
+  width: 90px;
+  height: 28px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 28px;
+  color: #28352f;
+}
```

Comment:
- [ ] Good job with following most of the styles from your chosen template on Figma. Please consider adding the required font-family `Inter` so that your design will match that of your chosen template 😃.